lib/publiccloud/instance.pm

```diff
@@ -65,7 +65,7 @@ sub run_ssh_command {
 
     my $ssh_cmd = sprintf('ssh %s "%s@%s" -- %s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd);
     $ssh_cmd = "timeout $timeout $ssh_cmd" if ($timeout > 0);
-    record_info('SSH CMD', $ssh_cmd);
+    record_info('SSH CMD', $ssh_cmd) unless ($rc_only);
```

Comment:
why we don't want to show command for case when you just need return code ? We already have `quiet` option maybe better to use it here instead ?

Reply:
I discarded this change. It should not be needed.